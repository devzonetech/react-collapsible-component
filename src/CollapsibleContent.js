import React from 'react'

export class CollapsibleContent extends React.Component {
    
    render() {

        const { className } = this.props;
        let addClassName = className ? className : '';

        return (
            <div className={' panel-accordion ' + addClassName}
                style={this.props.isExpanded ? {display: 'block'} : null}>
                {this.props.children}
            </div>)
    }
}