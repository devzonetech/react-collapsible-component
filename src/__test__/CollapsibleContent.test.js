import React from 'react'
import renderer from 'react-test-renderer'
import CollapsibleContent from '../CollapsibleContent'

describe('CollapsibleContent', () => {
    test('snapshot renders', () => {
        const component = renderer.create(<CollapsibleContent />)
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})