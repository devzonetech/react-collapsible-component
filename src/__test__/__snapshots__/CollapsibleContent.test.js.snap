// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollapsibleContent snapshot renders 1`] = `
<div
  className=" panel-accordion "
  style={null}
/>
`;
